### PR TITLE
DDCE-824: changes for P8A.1-P8A.3 Address Lookup

### PR DESCRIPTION
**Pull request by abhijeetardale**
# DDCE-824

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by abhijeetardale** [comment]
Linked PR please
https://github.com/hmrc/charities-registration-journey-tests/pull/84

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -75,13 +75,10 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, environmen
   def languageTranslationEnabled: Boolean = servicesConfig.getBoolean("features.welshLanguage")
```
Comment (anburoach) [inline app/config/FrontendAppConfig.scala:75]: Remove unused imports

File: conf/messages.en [hunk 2]
```diff
@@ -821,6 +821,26 @@ otherOfficialsNino.error.format = Enter a National Insurance number in the corre
 
 otherOfficialsNino.checkYourAnswersLabel = National Insurance number
 
+# Other  Official Address Messages
+# ----------------------------------------------------------
+otherOfficialAddress.lookupPage.title = What is the other official''s home address? - Officials and nominees
+otherOfficialAddress.lookupPage.heading = What is {0}''s home address?
+otherOfficialAddress.lookupPage.manualAddressLinkText = Other official''s home address does not have a UK postcode
+
+otherOfficialAddress.selectPage.title = What is the other official''s full address? - Officials and nominees
```
Comment (anburoach) [inline conf/messages.en:830]: this message should be "**What is the other official''s full home address? - Officials and nominees**"

File: conf/messages.en [hunk 3]
```diff
@@ -821,6 +821,26 @@ otherOfficialsNino.error.format = Enter a National Insurance number in the corre
...
+otherOfficialAddress.selectPage.heading = What is {0}''s full address?home
```
Comment (anburoach) [inline conf/messages.en:831]: this message should be "What is {0}''s full home address?"

File: conf/messages.en [hunk 4]
```diff
@@ -821,6 +821,26 @@ otherOfficialsNino.error.format = Enter a National Insurance number in the corre
...
+otherOfficialAddress.selectPage.editAddressLinkText = Enter other official''s address manually
```
Comment (anburoach) [inline conf/messages.en:832]: this message should be "**Enter other official''s home address manually** "

**Pull request merged**